src/pages/workspaces/workspace/WorkspaceContainer.test.ts

```diff
@@ -123,20 +149,23 @@ describe('WorkspaceContainer', () => {
     expect(screen.queryByRole('alert')).toBeNull();
   });
 
-  xit('polls for a workspace in the process of deleting and redirects when deleted', async () => {
+  it('does not poll for a workspace that is not deleting', async () => {
     // Arrange
-    const pollResponse: Response = new Response(null, { status: 404 });
-    const silentlyRefreshWorkspace = jest.fn().mockImplementation((errorHandling?: ErrorCallback) => {
-      if (errorHandling) {
-        errorHandling(pollResponse);
-      }
-    });
+
+    const mockDetailsFn = jest.fn().mockResolvedValue({ state: 'Deleting' });
```

Comment:
This has state `Deleting`, but the test point is about a workspace that is NOT in `Deleting`.

Reply:
I think the return value doesn't matter here-- you are just checking that the mock isn't called. I would just use a dummy implementation so it's clearer that the return state does not matter.

Reply:
You're definitely right: having the resolved value there makes this way more confusing than it has to be.